lib/reporters/dot.js

```diff
@@ -67,7 +67,7 @@ function Dot(runner) {
   });
 
   runner.once(EVENT_RUN_END, function() {
-    console.log();
+    process.stdout.write('\n');
```

Comment:
maybe we should leave this as-is until comes a time when we do this same PR against `process.stdout` and `process.stderr`.  what do you think?

Reply:
(I'm thinking we probably shouldn't do that)

Reply:
It was flagged as a pointless use of console.log to begin with and i agreed. Feels a good idea to have the reporters consistent while im here.

Reply:
IMO console.log should be used unless it can’t be. it does some queuing and stream-related stuff that prevents the chance for the output to be lost before the process ends. not a big deal right here since the epilogue will be output after.

Reply:
Can't find what you're in reference to [here](https://github.com/nodejs/node/blob/5e98f875b93b7556458a6a274b09a36875207d9d/lib/internal/console/constructor.js#L288).

Would appear to actually be a speed disadvantage using `console.log` for simple strings. For more complex strings, the format routine can be `require`'d and used, same as `console.log` does. Outside of use in browser, I find little use for `console.log` at all...